docs/en/sql-reference/aggregate-functions/index.md

```diff
@@ -72,3 +72,16 @@ FROM t_null_big
 │ 2.3333333333333335 │                 1.4 │
 └────────────────────┴─────────────────────┘
 ```
+
+Also you can use [Tuple](../data-types/tuple.md) to change NULL skipping behavior.
```

Comment:
```suggestion
Also you can use [Tuple](../data-types/tuple.md) to work around NULL skipping behavior. The a `Tuple` that contains only a `NULL` value is not `NULL`, so the aggregate functions won't skip that row because of that `NULL` value.
```